alvr/dashboard/src/dashboard/components/settings_controls/presets/builtin_schema.rs

```diff
@@ -321,7 +321,7 @@ pub fn microphone_schema(devices: Vec<String>) -> PresetSchemaNode {
 
 pub fn hand_tracking_interaction_schema() -> PresetSchemaNode {
     const HELP: &str = r"Disabled: hands cannot emulate buttons. Useful for using Joy-Cons or other non-native controllers.
-Separate trackers: create separate SteamVR devices for hand tracking. This is used for VRChat.
+SteamVR Input 2.0: create separate SteamVR devices for hand tracking.
```

Comment:
I believe "SteamVR Input 2.0" has clearer intentions. that feature both spawn 2 pairs of controllers (2 as hand trackers) and at the same time changes the skeletal tracking from partial to either full or estimated. this is not clear to summarize in one name and "SteamVR Input 2.0" more clearly tells people when they want to use it, because games themselves advertise being compatible with "SteamVR Input 2.0"